### ISSUE TITLE
Media sync between pre-Cloud and Cloud infrastructure

### ISSUE THREAD

**Issue by rjackson** [tfwiki/deployment#10]
If we want two way sync, [Unison](http://www.cis.upenn.edu/~bcpierce/unison/) should be able to help us with this. It should allow us to sync our `mediawiki-images` volume (Google Compute disk backed NFS PersistentVolume) with the Valve-infrastructure media folder (over SSH tunnel).

Otherwise, rsync will suffice.

**Comment by rjackson** [comment]
I've synced the files from the Valve-hosted Wiki to the Cloud production instance by running the following command directly on the `mediawiki` box:

```
rsync -av --delete -e "ssh -p VALVE_SSH_PORT -A -t VALVE_USER@VALVE_ADMIN_BOX -A -t VALVE_WEB_BOX" :/valve/shared/wiki.teamfortress.com/images/ /var/www/html/w/images
```

Note having to connect to a web box via the admin box: The wiki's NFS hasn't been mounted on the admin box since it was spun up afresh.

We can wrap this up into a Kubernetes job once the new SSH keypair for syncing files has been granted access to the admin box.

**Comment by rjackson** [comment]
See also https://kubernetes.io/docs/concepts/configuration/secret/#use-case-pod-with-ssh-keys

**Comment by rjackson** [comment]
Implemented with 0df44e60c1747c14c69e180271aafd882f870283
